redisinsight/api/src/modules/browser/redisearch/redisearch.service.ts

```diff
@@ -269,6 +271,34 @@ export class RedisearchService {
     }
   }
 
+  public async deleteIndex(
```

Comment:
it seems like there is no appropriate unit test added for this method

Reply:
Thanks for the reminder, I went for the integration tests and forgot about the unit tests.

Now, we have coverage for this method as well.